lib/config/account/index.js

```diff
@@ -0,0 +1,22 @@
+module.exports = accountConfig
+
+var defaultsDeep = require('lodash').defaultsDeep
+
+function accountConfig (state, callback) {
+  var usersDb = new state.config.PouchDB(state.db.authenticationDb)
+  usersDb.installUsersBehavior()
```

Comment:
Hmm I remember struggling myself to decide what the right place is for `pouchdb-users`. I think as we pass in the `PouchDB` constructor, I feel like we should apply all plugins beforehand. On the other side we can apply it within `@hoodie/server`, too ... maybe that’ll make things simpler? I’ll give it another thought, thanks!